docs/index.html

```diff
@@ -15,56 +16,23 @@
   <meta property="og:description" content="The Dafny Programming and Verification Language and tools.">
   <meta property="og:image" content="image.png">
 
-  <link rel="icon" href="dafny-favicon.ico">
-  <link rel="icon" href="dafny-favicon.svg" type="image/svg+xml">
+  <link rel="icon" href="images/dafny-favicon.png" type="image/png">
+  <link rel="icon" href="images/dafny-favicon.svg" type="image/svg+xml">
 
 </head>
 
-<style>
-body {
-  background-color: light-blue;
-}
-
-h1 {
-  color: blue;
-  text-align: center;
-  background-color: #fceb6c
-}
-
-p {
-  font-size: 18px
-}
-
-td {
-  font-size: 18px
-}
-
-li {
-  font-size: 18px
-}
-
-img {
-    display: block;
-    margin-left: auto;
-    margin-right: auto;
-}
-
-</style>
-
 <body>
 
 <div>
-<h1>Dafny<br>
-<font size="-1">
-Development version
-</font>
-</h1>
+<h1>Dafny<br><p>
+<!-- VERSION INFO HERE-->Development version
```

Comment:
Will this be the target of some pre-processing script then?

Reply:
Yes.